kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -493,18 +493,10 @@
     },
     $trs: {
       createAccountAction: 'Create an account',
-      kolibri: 'Kolibri',
       poweredByKolibri: 'Powered by Kolibri',
       whatsThis: "What's this?",
       oidcGenericExplanation:
         'Kolibri is an e-learning platform. You can also use your Kolibri account to log in to some third-party applications.',
-      oidcSpecificExplanation:
-        "You were sent here from the application '{app_name}'. Kolibri is an e-learning platform, and you can also use your Kolibri account to access '{app_name}'.",
```

Comment:
let's leave this string in for now in case we need it eventually

Reply:
Not sure why it doesn't say this is outdated, but I've added it back along with a comment disabling the rule for that line only and a comment noting that we are keeping the unused string in case we use it in the future. @indirectlylit